### DISCUSSION TITLE
Dropdown component

### DISCUSSION THREAD

**Discussion by kandrelczyk** [thaw-ui/thaw#205] [General]
Hi,

I'm missing a nice dropdown component like this one: https://www.naiveui.com/en-US/os-theme/components/dropdown

I use drawer for the main menu and I need dropdown like component for a secondary menu, similar to this:

![image](https://github.com/thaw-ui/thaw/assets/13573289/438f7778-4ab4-4565-81a9-0e5ef1a120bb)

Currently we can almost get something like this with Popover and Menu combination but they don't work well together. Popover's background color and menu's font color don't have enough contrast (not to mention paddings and the unnecessary arrow). 

![image](https://github.com/thaw-ui/thaw/assets/13573289/28a28517-c854-41a5-9951-58327cf0325b)

Select component looks right but it's function is completely different. 

What do you think is the best way to get the Dropdown component? I think the confusing part for me is that it should behave like Popover with Menu inside but it should look like Select component. Should just be a completely new component following the design of Select but working like Popover? The menu inside Dropdown should also be a new component? Or do you think it's possible to reuse parts of the existing components?

**Comment by lizidev** [comment]
I think we can just develop a new Dropdown component. Reusing existing components will only increase coupling.

**Comment by kandrelczyk** [comment]
Ok, I'm happy to start working on it if you haven't already.

**Comment by lizidev** [comment]
I haven't started writing yet, welcome PR.
